.github/workflows/go.yaml

```diff
@@ -20,6 +20,8 @@ jobs:
         id: go
       - name: Generated code
         run: make gen
+      - name: Install Protoc
+        uses: arduino/setup-protoc@v1
```

Comment:
arduino created this action?  No other action?? :unamused:

Reply:
![image](https://user-images.githubusercontent.com/8110/103221489-77f68c80-492b-11eb-8c8a-db13e695f6a7.png)
didn't find anything better in the github marketplace